lisk/EXCEPTIONS.md

```diff
@@ -0,0 +1,231 @@
+## Exceptions
```

Comment:
file name should be in small letters? I think no reason to be in capital

Reply:
We use capital names for all `.md` files in the project. 

https://github.com/LiskHQ/lisk-sdk/tree/development/docs
https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/README.md

any many others, so why kept this one as capital case as well.